### ISSUE TITLE
"Insert page" creates a duplicate of the page you were viewing. It is a bit counterintuitive to understand how to create a blank page.

### ISSUE THREAD

**Issue by All3xJ** [xournalpp/xournalpp#2469]
**Is your feature request related to a problem? Please describe.**
"Insert page" creates a duplicate of the page you were viewing. It is a bit counterintuitive to understand how to create a blank page.

**Describe the solution you'd like**
It would be better if, by default, the template of the page is set to BLANK page, not to "copy current page background". And also, it should be horizontal, not vertical, to my view.

**Comment by rolandlo** [comment]
I don't agree with the solution you suggest. Unless you annotate a pdf it is more intuitive to copy the page background than to insert a blank page. Also I don't think that pages should be landscape by default.

**Comment by All3xJ** [comment]
> I don't agree with the solution you suggest. Unless you annotate a pdf it is more intuitive to copy the page background than to insert a blank page. Also I don't think that pages should be landscape by default.

As a new user of Xournal++, I can just say that I found myself lost at the beginning when I was trying to create a new page and it copied the previous one.

**Comment by Technius** [comment]
By "copied the previous one" do you mean it copied everything on the previous page, including annotations?

**Comment by All3xJ** [comment]
> By "copied the previous one" do you mean it copied everything on the previous page, including annotations?

no, not the annotations.

**Comment by Technius** [comment]
Can you provide some details on your use case? The defaults are set up for note taking purposes, which lined paper works well for most of the time.

I also think we need to improve the documentation of the page background feature--another thing to do for the new website.

**Comment by All3xJ** [comment]
> Can you provide some details on your use case? The defaults are set up for note taking purposes, which lined paper works well for most of the time.

If I want to edit an existing PDF, and I want to add a page after a certain one, if I click on "Insert page after", I'd expect to have a new blank page, not a duplicate of the previous one. I found it a bit counterintuitive, only this.

**Comment by Rmano** [comment]
> If I want to edit an existing PDF, and I want to add a page after a certain one, if I click on "Insert page after", I'd expect to have a new blank page, not a duplicate of the previous one. I found it a bit counterintuitive, only this.

This surprised me also the first time. I use `xournalpp` a lot to correct exams and lab reports, and sometimes I want to add a page to make additional comments --- it's not a big problem (you can change that page background later). 

On the other hand, if you have a nifty background PDF format for your notes, repeating the background is exactly what you need --- and given that there is no option to directly do go back from say, a lined background to "a PDF page background which is the same as the previous one", I vote to keep the current behavior.
